src/pyOpenMS/create_cpp_extension.py

```diff
@@ -106,15 +106,21 @@
 
 
 if iswin:
-    # fix for broken library names in release 1.11:
-    for p in glob.glob(os.path.join(OPEN_MS_CONTRIB_BUILD_DIR,
-                                    "lib",
-                                    "libboost_math_*mt.lib")):
-
-        if "vc90" in p:
-            continue
-        new_p = p.replace("-mt.lib", "-vc90-mt-1_52.lib")
-        shutil.copy(p, new_p)
+    for libname in ["math", "regex"]:
```

Comment:
maybe @uweschmitt can comment on this but it may be a Python thing ? I dont think they are hardcoded on our side but possible from Cython/Python side?